tests/example_app/example/about/our_team/static/app.css

```diff
@@ -0,0 +1,3 @@
+p {
+    color: purple;
```

Comment:
Can you use 2 spaces instead of 4 and set `color: #00ff00;`?

I have nothing against purple but the other colors were red and blue. Using green here completes the RGB trilogy.